rcloud.support/R/ocaps.R

```diff
@@ -137,7 +138,7 @@ unauthenticated.ocaps <- function(mode, compute)
       signal_to_compute = make.oc(.signal.to.compute),
       help = compute$help,
       get_thumb = compute$get_thumb,
-
+      get_fork_count = compute$get_fork_count,
```

Comment:
We should revisit if these need to be run from compute process or if they can be moved to the control process. If it works for now, that's fine, but I think it's generally better to put them on control if possible. (They will wait for cells to execute, and I think we eventually want to move all RCS access to control for security reasons.)